scripts/kind_up.sh

```diff
@@ -23,4 +23,5 @@ if [ -n "$EKCP_HOST" ]; then
     curl -d "name=${CLUSTER_NAME}" -X POST http://$EKCP_HOST/new
 else
     ./kind create cluster --config kind-config.yaml --name=${CLUSTER_NAME}
+    cp $(./kind get kubeconfig-path --name="$CLUSTER_NAME") kubeconfig
```

Comment:
Are we dropping the `kubeconfig` target ? mind that this mechanism is also used by the `dind` one (bootstrap SCF within the docker image)

Reply:
This seems a dup, it would collide with the `make kubeconfig` target that does the same (just chain it where you need it)

Reply:
Thinking twice: we could drop the `docker-kubeconfig` target by fusing it with `kubeconfig`, in that way we can drop also the `dind` target - they just need a switch (maybe an ENV var?) that can be pinned as default from the Dockerfile

Reply:
In my mind, the canonical place to obtain a kubeconfig is `buildir/kubeconfig`; either you have it there because the k8s cluster deployment succeeded (be it kind, caasp4, caasp3, etc), or you don't have it.

Therefore I would remove the `make kubeconfig`, and keep the `make docker-kubeconfig` to be used with dind.

I can see the benefit of moving into the future to a makefile that triggers different deployments and make targets depending on an env var PLATFORM={kind,dind,caasp4os,caasp4bare,gke,eks} etc.

Reply:
Just please mind https://github.com/SUSE/bkindwscf/commit/8fa83463d628397ce98961eea46ce3f82a1b5ea3 while doing this, we found other "ways" to get a kubeconfig dynamically :)

Reply:
Removed the kubeconfig acquisition from scf_install. Since `make kind` and `make dind` keep having a `kubeconfig` target, everything is as it was before